sceptre/bridge/templates/bridge.yaml

```diff
@@ -313,10 +296,6 @@ Outputs:
         - !GetAtt AWSS3AndroidAppBucket.DomainName
     Export:
       Name: !Sub '${AWS::StackName}-AndroidAppBucketSecureURL'
-  VpnSecurityGroup:
```

Comment:
this is referenced in https://github.com/Sage-Bionetworks/BridgeServer2-infra/blob/develop/templates/bridgeserver2-db.yaml#L47 and https://github.com/Sage-Bionetworks/BridgeServer2-infra/blob/develop/templates/bridgeserver2.yaml

Reply:
I think these are coming from BridgeServer2-vpc stack (based on vpc.yaml which also creates a VpnSecurityGroup). This one would be referenced as bridge-VpnSecurityGroup would it not?

Reply:
sorry, you are correct.